sys/include/can/isotp.h

```diff
@@ -7,12 +7,11 @@
  */
 
 /**
- * @ingroup     can
- * @defgroup    isotp ISOTP
+ * @defgroup    sys_can_isotp ISOTP
```

Comment:
Can you please pull up the wording from the brief up here (without the ISO number), but leave the `@brief` as is? ISOTP isn't really saying anything to me ;-).